pages/settings/profile.tsx

```diff
@@ -257,22 +302,20 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
     return { redirect: { permanent: false, destination: "/auth/login" } };
   }
 
-  const user = await prisma.user.findFirst({
-    where: {
-      email: session.user.email,
-    },
-    select: {
-      id: true,
-      username: true,
-      name: true,
-      email: true,
-      bio: true,
-      avatar: true,
-      timeZone: true,
-      weekStart: true,
-      hideBranding: true,
-    },
-  });
+  const user = await whereAndSelect(prisma.user.findFirst, {
```

Comment:
Codacy has a fix for the issue: [Replace `prisma.user.findFirst,` with `⏎····prisma.user.findFirst,⏎···`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7734611)
```suggestion
  const user = await whereAndSelect(
```